docs/docs/concepts/redirects.mdx

```diff
@@ -1,9 +1,146 @@
+with the Login/Registration `Default Redirect Url` and then the
+`Global Redirect Url`. With a path it will use the closest relative as the base
+url as described above with the `Default Redirect Url`.
+
+![Browser Redirects Url Management](../images/browser-redirects/browser-redirects-url-management.png)
+
+All the relative urls will be updated with their respective base url. As shown
+in the example below, the Login `Default Redirect Url` has been set to use a
+relative url and thus the base url will be the `Global Redirect Url`
+(https://myapp.com). This will then set the Login `Default Redirect Url` to
+https://myapp.com/after/login and the `Post-Password` to
+https://myapp.com/after/login/password.
+
+![Browser Redirects Url Management Transformed](../images/browser-redirects/browser-redirects-url-management-2.png)
+
+### Verification, Recovery and Logout
+
+:::warning
+
+The `Post-Recovery` redirect is currently not supported and will automatically
+redirect the user to the `Settings UI` url. This implies that you should
+currently rely on the `Post-Settings` redirect url for `Post-Recovery` flows as
+well.
+
+:::
+
+Each of these flows have only a single field, the `Default Redirect Url`. As
+with the Login, Registration and Settings configurations, the
+`Default Redirect Url` can be a FQDN or a path.
+
+![Browser Redirects Url Management Transformed](../images/browser-redirects/browser-redirects-url-management-3.png)
+
+## Resetting/Updating Flows
+
+A redirect can easily be updated or reset by either changing the current
+redirects value or deleting the entry and clicking the "Update" button. All the
+flows can be reset at once by pressing the "Reset Flows" button which will
+prompt a confirmation box. Resetting all flows will reset all the fields back to
+the default value which is the Ory Cloud
+[Managed User Interface](./managed-ui.mdx).
+
+Once a flow has been reset or updated it will take effect immediately on **new**
+flows. Old flows that have not expired yet will not be updated.
+
+![Browser Redirects Url Reset All](../images/browser-redirects/browser-redirects-reset-all.png)
+
+## Troubleshooting Guide
+
+### Allow List
+
+The `Allow List` requires a valid url to be set with a scheme (http or https).
+
+![Browser Redirects Url Error Message](../images/browser-redirects/browser-redirects-error-messages-1.png)
+
+### Post Flow
+
+Any of the Post-flow redirects require a valid url with scheme (http or https)
+or path. An example of a valid url is https://www.google.com.
+
+![Browser Redirects Url Error Message Flows](../images/browser-redirects/browser-redirects-error-messages-flows.png)
+
+### Domain Denied
+
+URLs matching any of the denied urls, such as the Ory domain will automatically
+be denied. This deny list can be extended in future to prevent any abuse on the
+Ory Cloud.
+
+![Browser Redirects Url Error Message](../images/browser-redirects/browser-redirects-error-messages-2.png)
```

Comment:
```suggestion
The `Allow List` requires a valid URL to be set with a scheme (http or https).

![Browser Redirects URL Error Message](../images/browser-redirects/browser-redirects-error-messages-1.png)

### Post Flow

Any of the Post-flow redirects require a valid URL with scheme (http or https)
or path. An example of a valid URL is https://www.google.com.

![Browser Redirects URL Error Message Flows](../images/browser-redirects/browser-redirects-error-messages-flows.png)

### Domain Denied

URLs matching any of the denied URLs, such as the Ory domain will automatically
be denied. This deny list can be extended in future to prevent any abuse on the
Ory Cloud.

![Browser Redirects URL Error Message](../images/browser-redirects/browser-redirects-error-messages-2.png)
```